spec/views/category_spec.rb

```diff
@@ -0,0 +1,29 @@
+require 'rails_helper'
+
+RSpec.describe 'groups page', type: :feature do
+  before :each do
+    @user = User.create(
+      name: 'John',
+      email: 'john@example.com',
+      password: 'password'
+    )
+
+    visit new_user_session_path
+    fill_in 'Email', with: @user.email
+    fill_in 'Password', with: @user.password
+    click_button 'Log in'
+  end
+
+  it 'should display the category page' do
+    expect(page).to have_selector('div', text: 'CATEGORY')
+  end
+
+  it 'should display add new category button' do
+    expect(page).to have_link('Add a new category')
+  end
+
+  it 'should display the category form' do
+    click_link 'Add a new category'
+    expect(page).to have_selector('div', text: 'NEW CATEGORY')
+  end
+end
```

Comment:
- [ ] some of the tests are failing please make sure to fix them in order to complete the requirements 
<img width="973" alt="Screenshot 2023-04-27 at 21 06 32" src="https://user-images.githubusercontent.com/61790587/234979556-9c376d26-84ee-4155-a730-f58698afcbc4.png">